cmake/Modules/FindSYCLToolkit.cmake

```diff
@@ -49,12 +50,28 @@ find_file(
   )
 
 # Find SYCL library fullname.
-find_library(
-  SYCL_LIBRARY
-  NAMES sycl
-  HINTS ${SYCL_LIBRARY_DIR}
-  NO_DEFAULT_PATH
-)
+if(WIN32)
+  # TODO: we can drop this workaround once an open-source release
```

Comment:
which issue and which workaround? please, update comment to give a hint to other developers on what is being awaited. If there is a public issue filed somewhere at github, I suggest to add a link.

Reply:
Thanks, updated comment after offline sync with compiler team.